style.css

```diff
@@ -1 +1,254 @@
+
+.hamburger-menu {
+    width: 6em;
+    height: 7em;
+}
+
+.hamburger-menu-item {
+    font-size: small;
+    background-color: #CDF3AC;
+    margin: 0.5em 0;
+    font-weight: bold;
+    padding: 0.25em;
+    text-align: right;
+}
+
+.large-nav {
+    display: none;
+}
+
+
+.slogan {
+    font-size: medium;
+    font-weight: bold;
+    width: 13em;
+    height: 5em;
+    text-align: right;
+    margin-bottom: 2em;
+    display: flex;
+    align-items: end;
+}
+
+
+.header-right {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    margin: 1em 0 0 0;
+}
+
+
+h1 {
+    font-weight: bold;
+    margin-top: 2em;
+}
+
+.site-description {
+    font-weight: bold;
+    text-align: justify;
+}
+
+.hero-section {
+    width: auto;
+    height: auto;
+    background-color: #E7E9E8;
+}
+
+#hero-img {
+    width: 100%;
+    height: 100%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.cake-display {
+    margin: 27px 0;
+    display: grid;
+    gap: 20px;
+    /* grid-template-columns: repeat(auto-fit, minmax(170px, 1fr));
+    grid-template-rows: repeat(2, auto); */
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto;
+}
+
+/* each box is width=201 and heith=125 */
+
+img {
+    width: 100%;
+    height: 100%;
+}
+
+.footer-section {
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    height: auto;
+    
+}
+.side-footer {
+    display: flex;
+    flex-direction: column;
+    height: 7rem;
+    width: 14rem;
+    margin: 2rem 1rem 1rem 1rem;
+}
+
+.side-footer-item {
+    margin: 0.5rem;
+    background-color: #CDF3AC;
+    font-size: large;
+    font-weight: bold;
+    padding: 0.25em;
+}
+
+.center-footer {
+    align-items: end;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #F1D8D5;
+    margin-bottom: 2rem;
+    padding: 0.25rem;
+}
+
+
+@media screen and (min-width: 540px) {
+    body {
+        background-color: #d7e4ca;
+    }
+    .center-footer {
+    background-color: #d7e4ca;
+}
+}
+
+@media screen and (min-width: 900px) {
+    body {
+        background-color: #f3e3bc;
+    }
+    .cake-display {
+        grid-template-columns: 1fr 1fr 1fr 1fr;
+        grid-template-rows: auto;
+    }
+    .hamburger-menu {
+        display: none;
+    }
+
+    .large-nav {
+        display: flex;
+        flex-direction: row;
+        justify-content: right;
+        background-color: #F4ACB7;
+        height: auto;
+        width: auto;
+        margin-top: 2rem;
+        margin-bottom: 3rem;
+        padding: 0.25em 1em;
+    }
+    
+    .large-nav-item {
+        text-align: center;
+        font-size: large;
+        font-weight: bold;
+        background-color: #CDF3AC;
+        margin: 1em;
+        justify-content: space-evenly;
+        font-weight: bold;
+        padding: 0.5em;
+        width: 9rem;
+
+    }
+    
+
+    .slogan {        
+        display: flex;
+        background-color: #F4ACB7;
+        font-size: x-large;
```

Comment:
I think we shouldn't say `font-size: x-large`. Look here https://www.w3schools.com/css/css_font_size.asp it is done as a numeric unit